core/src/main/java/org/testng/xml/XmlInclude.java

```diff
@@ -61,6 +72,12 @@ public void setName(String name) {
     return m_invocationNumbers;
   }
 
+  public void addInvocationNumbers(List<Integer> invocationNumberList) {
+    m_invocationNumbers.addAll(invocationNumberList);
+    // Deleting of duplicates numbers
+    m_invocationNumbers = m_invocationNumbers.stream().distinct().collect(Collectors.toList());
```

Comment:
Just move `m_invocationNumbers` from list to set and update getInvocationNumbers() for a set->list transformation.

Reply:
Thanks for your feedback. done